### DISCUSSION TITLE
application starts with document open

### DISCUSSION THREAD

**Discussion by g33kdad** [KDE/ghostwriter#676] [Q&A]
Hi,

I have ghostwriter installed in the debian 10 container on my chromebook. for some reason, whenever i launch the application, the very first document i ever opened is open in the window. the file is saved, but i can't find any way to close it. i can open other documents or create new ones, but that first document is always open in the background... any thoughts?

Cheers,
Matty

**Comment by Hermann12** [comment]
File -> New , is this what you search for? ( I use Win10)
